homeassistant/components/openai_conversation/__init__.py

```diff
@@ -70,6 +70,11 @@ def attribution(self):
         """Return the attribution."""
         return {"name": "Powered by OpenAI", "url": "https://www.openai.com"}
 
+    @property
+    def supported_languages(self) -> list[str]:
+        """Return a list of supported languages."""
+        return ["smurfish"]
```

Comment:
Could we add some special value to the language utility which means any language, and will match against all languages?

Reply:
`from homeassistant.const import MATCH_ALL`